### ISSUE TITLE
"Zoom in to explore isolines" label covers the bottom bar

### ISSUE THREAD

**Issue by not-available-username** [organicmaps/organicmaps#2556]
on iOS the label that says "Zoom in to explore isolines" is covering the bottom bar, should be moved upwards

![File 14-05-22, 14 04 42](https://user-images.githubusercontent.com/92886370/168425114-c03e1109-8e1b-44cb-8e37-197dae1a1c77.jpeg)

**Comment by not-available-username** [comment]
Could someone check this? I know that there are many bugs, but on iOS this is quite annoying and it seems not too hard to fix. Thank you so much!

**Comment by varjas** [comment]
Checked this on the versions below and do not see any label for this at various zoom levels.

Organic Maps: 2023.09.22-20
iOS: 16.6

**Comment by not-available-username** [comment]
> Checked this on the versions below and do not see any label for this at various zoom levels.
> 
> Organic Maps: 2023.09.22-20 iOS: 16.6

Do you have the isolines enabled? It happens only when you open the app for the first time, try to close the app in background.

**Comment by varjas** [comment]
Yes isolines are enabled. It does show up for ~3 seconds if the app is closed in the background and reopened.

**Comment by not-available-username** [comment]
> Yes isolines are enabled. It does show up for ~3 seconds if the app is closed in the background and reopened.

that's exaclty the bug, the label is misaligned and it covers the icons, it should be places above the bottom bar.
